bindings/java/src/main/com/apple/foundationdb/tuple/ByteArrayUtil.java

```diff
@@ -418,5 +418,94 @@ public static String printable(byte[] val) {
 		return s.toString();
 	}
 
+	/**
+	 * Converts the appropriate bytes to Object.
+	 *
+	 * @param bytes The encoded elements.
+	 * @param num Converts the element at {@code num} to Object.
```

Comment:
If we encode 3 objects, with "num" we can specify which object we want to decode (0, 1 or 2). Instead of "num" we can also use "skip" variable in which we should specify how many bytes to skip (the starting position of "bytes" will be skip + 1). Or we can use "num" and "skip" together.

Reply:
That makes sense; it just needs to be documented. I would also prefer it to be `index` and then the documentation can explain that it is the index of elements in the decoded tuple for clarity.